packages/shared-components/src/components/HelpCallout/HelpCallout.js

```diff
@@ -75,10 +107,7 @@ class HelpCallout extends React.Component {
 
     return (
       <Manager>
-        <div style={{ display: 'flex' }}>
-          {children}
```

Comment:
I actually wrote the children outside the reference here for a reason; the callout should be centered on the icon and not the text when you have the icon + text format.

Reply:
But it centered on the icon, not the whole text, this is why you have `= ({ ref }) =>` in the `<Reference />`

Reply:
Oh right. Ignore me... 🐵